submissions/UA-girl/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+const IMG_CLASS_NAME = 'card-image';
+const CARD_DESCRIPTION_CLASS_NAME = 'card-description';
+const CARD_TITLE_CLASS_NAME = 'card-header';
+const CARD_TEXT_CLASS_NAME = 'card-text';
+const CARD_LINK_CLASS_NAME = 'link-wiki';
+const catalogItemsList = [
```

Comment:
I'd recommend you renaming this variable. It doesn't give a context. I think it would be better to use cats in this variable name as this data connected to the information about the cats. Maybe `catsDataList` would be better.